Packs/QutteraWebsiteMalwareScanner/Integrations/QutteraWebsiteMalwareScanner/README.md

```diff
@@ -0,0 +1,140 @@
+Quttera Website Malware Scanner
```

Comment:
Just making sure, did you run the generate-docs command after all your fixes? the readme file is up to date?